2.0/DamagedHelmet/glTF/DamagedHelmet.gltf

```diff
@@ -1,204 +1,242 @@
-                    "indices" : 0,
-                    "material" : 0
-                }
-            ]
-        }
-    ],
-    "nodes" : [
-        {
-            "mesh" : 0,
-            "name" : "node_damagedHelmet_-6514",
-            "rotation" : [
-                0.7071068286895752,
-                0.0,
-                -0.0,
-                0.7071068286895752
-            ]
+        "metallicRoughnessTexture": {
+          "index": 1
         }
-    ],
-    "samplers" : [
-        {}
-    ],
-    "scene" : 0,
-    "scenes" : [
-        {
-            "name" : "Scene",
-            "nodes" : [
-                0
-            ]
+      },
+      "normalTexture": {
+        "index": 4
+      },
+      "occlusionTexture": {
+        "index": 3
+      },
+      "emissiveTexture": {
+        "index": 2
+      },
+      "emissiveFactor": [
+        1.0,
+        1.0,
+        1.0
+      ],
+      "name": "Material_MR"
+    }
+  ],
+  "meshes": [
+    {
+      "primitives": [
+        {
+          "attributes": {
+            "NORMAL": 2,
+            "POSITION": 1,
+            "TEXCOORD_0": 3
+          },
+          "indices": 0,
+          "material": 0
         }
-    ],
-    "textures" : [
-        {
-            "sampler" : 0,
-            "source" : 0
-        },
-        {
-            "sampler" : 0,
-            "source" : 1
-        },
-        {
-            "sampler" : 0,
-            "source" : 2
-        },
-        {
-            "sampler" : 0,
-            "source" : 3
-        },
-        {
-            "sampler" : 0,
-            "source" : 4
+      ],
+      "name": "mesh_helmet_LP_13930damagedHelmet"
+    }
+  ],
+  "nodes": [
+    {
+      "name": "node_damagedHelmet_-6514",
+      "rotation": [
+        0.7071068286895752,
+        0.0,
+        -0.0,
+        0.7071068286895752
+      ],
+      "mesh": 0
+    }
+  ],
+  "samplers": [
+    {}
+  ],
+  "scene": 0,
+  "scenes": [
+    {
+      "nodes": [
+        0
+      ],
+      "name": "Scene"
+    }
+  ],
+  "textures": [
+    {
+      "source": 0,
+      "sampler": 0
+    },
+    {
+      "source": 1,
+      "sampler": 0
+    },
+    {
+      "sampler": 0,
+      "source": 2
+    },
+    {
+      "sampler": 0,
+      "source": 3
+    },
+    {
+      "source": 4,
+      "sampler": 0
+    }
+  ],
+  "extensions": {
+    "KHR_xmp": {
+      "@context": {
+        "dc": "http://purl.org/dc/elements/1.1/",
+        "model3d": "https://schema.khronos.org/"
+      },
+      "packets": [
+        {
+          "dc:description": {
+            "en-us": "The original model was built on an early draft of glTF 2.0 that did not become final. This new model has been imported and re-exported from Blender to bring it into alignment with the final release glTF 2.0 specification."
+          },
+          "dc:format": "model/gltf-binary",
+          "dc:language": [
+            "en"
+          ],
+          "dc:relation": [
+            "https://www.khronos.org/",
```

Comment:
What's the relation to Khronos here?  (Other than we've over-used this model for a long time...)  Should this instead be a link to `https://sketchfab.com/theblueturtle_`?